declarations/ValidationError.d.ts

```diff
@@ -1,22 +1,11 @@
 export default ValidationError;
 export type JSONSchema6 = import('json-schema').JSONSchema6;
 export type JSONSchema7 = import('json-schema').JSONSchema7;
-export type Schema =
-  | (import('json-schema').JSONSchema4 & import('./validate').Extend)
-  | (import('json-schema').JSONSchema6 & import('./validate').Extend)
-  | (import('json-schema').JSONSchema7 & import('./validate').Extend);
-export type ValidationErrorConfiguration = {
-  name?: string | undefined;
-  baseDataPath?: string | undefined;
-  postFormatter?: import('./validate').PostFormatter | undefined;
-};
-export type PostFormatter = (
-  formattedError: string,
-  error: import('./validate').SchemaUtilErrorObject
-) => string;
-export type SchemaUtilErrorObject = import('ajv').ErrorObject & {
-  children?: import('ajv').ErrorObject[] | undefined;
-};
+export type Schema = import('./validate').Schema;
+export type ValidationErrorConfiguration =
+  import('./validate').ValidationErrorConfiguration;
+export type PostFormatter = import('./validate').PostFormatter;
+export type SchemaUtilErrorObject = import('./validate').SchemaUtilErrorObject;
```

Comment:
Can you test it manually? Don't want to break types, sometimes ts have small problems

Reply:
It looks good, 
In `./validate` -
https://github.com/webpack/schema-utils/blob/cc29077b6ac862ab96e0f39145e807310472b4ad/declarations/validate.d.ts#L11-L14

https://github.com/webpack/schema-utils/blob/cc29077b6ac862ab96e0f39145e807310472b4ad/declarations/validate.d.ts#L22-L26

https://github.com/webpack/schema-utils/blob/cc29077b6ac862ab96e0f39145e807310472b4ad/declarations/validate.d.ts#L18-L21

https://github.com/webpack/schema-utils/blob/cc29077b6ac862ab96e0f39145e807310472b4ad/declarations/validate.d.ts#L15-L17